frontend/src/lib/components/PropertyFilters/PropertyFilters.js

```diff
@@ -1,44 +1,93 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { PropertyFilter } from './PropertyFilter'
 import { Button } from 'antd'
 import { useValues, useActions } from 'kea'
 import { propertyFilterLogic } from './propertyFilterLogic'
+import { Popover, Row } from 'antd'
+import { CloseButton } from '../../utils'
 
-export function PropertyFilters({ endpoint, propertyFilters, className, style, onChange, pageKey }) {
+const formatFilterName = str => {
```

Comment:
This does almost the same as the `operatorMap` in PropertyFilter. Maybe worth combining the 2?